.github/label-actions.yml

```diff
@@ -0,0 +1,34 @@
+# When `devnet-e2e-test` is added, also assign `devnet` to the PR.
+devnet-e2e-test:
+  prs:
+    label:
+      - devnet
+
+# When `devnet-e2e-test` is removed, also delete `devnet` from the PR.
+-devnet-e2e-test:
+  prs:
+    unlabel:
+      - devnet
+
+# When `devnet` is added, also assign `push-image` to the PR.
+devnet:
+  prs:
+    label:
+      - push-image
+
+# When `devnet` is removed, also delete `push-image` from the PR.
+-devnet:
+  prs:
+    unlabel:
+      - push-image
+      - devnet-e2e-test
+
+# Let the developer know that they need to push another commit after attaching the label to PR.
```

Comment:
Can you add a Makefile target to push an empty commit and mention it in the comment?